crates/sdk/src/network/mod.rs

```diff
@@ -0,0 +1,120 @@
+use serde::{Deserialize, Serialize};
+use std::env;
+
+pub mod prover;
+
+#[derive(Debug, Clone)]
+pub struct NetworkClientCfg {
+    pub endpoint: Option<String>,
+    pub ca_cert_path: Option<String>,
+    pub cert_path: Option<String>,
+    pub key_path: Option<String>,
+    pub domain_name: Option<String>,
+    pub proof_network_privkey: Option<String>,
+}
+
+impl Default for NetworkClientCfg {
+    fn default() -> Self {
+        let endpoint =
+            Some(env::var("ENDPOINT").unwrap_or("https://152.32.186.45:20002".to_string()));
+        let domain_name = Some(env::var("DOMAIN_NAME").unwrap_or("stage".to_string()));
+        let proof_network_privkey = Some(
+            env::var("ZKM_PRIVATE_KEY").expect("ZKM_PRIVATE_KEY must be set for remote proving"),
+        );
+        let current_dir = env::current_dir().expect("Failed to get current directory");
+        let ca_cert_path = Some(current_dir.join("tool/ca.pem").to_string_lossy().to_string());
+        let cert_path =
+            Some(env::var("CERT_PATH").expect("CERT_PATH must be set for remote proving"));
+        let key_path = Some(env::var("KEY_PATH").expect("KEY_PATH must be set for remote proving"));
+
+        Self { endpoint, ca_cert_path, cert_path, key_path, domain_name, proof_network_privkey }
+    }
+}
+
+#[derive(Debug, Deserialize, Serialize, Clone)]
+pub struct ProverInput {
+    pub elf: Vec<u8>,
+    pub public_inputstream: Vec<u8>,
+    pub private_inputstream: Vec<u8>,
+    pub shard_size: u32,
+    // pub execute_only: bool,
+    pub composite_proof: bool,
+    pub receipt_inputs: Vec<Vec<u8>>,
+    pub receipts: Vec<Vec<u8>>,
+    pub asset_url: String,
+    pub use_network_vk: bool,
+    pub proof_results_path: String,
+}
+
+impl Default for ProverInput {
+    fn default() -> Self {
```

Comment:
Rename to `from_env`.

Reply:
The default is good for now, since we have no other ways to initialize the ProverInput.

Reply:
ok